tensorflow_addons/seq2seq/attention_wrapper.py

```diff
@@ -786,11 +782,14 @@ def _calculate_attention(self, query, state):
 
     def get_config(self):
         config = {
-            "units": self.units,
-            "normalize": self.normalize,
-            "probability_fn": self.probability_fn_name,
+            "units":
```

Comment:
I guess this is reformatted by the auto format, can we change it back since the existing one is more readable? Thanks.

Reply:
Agree, I disabled the auto format for those configs